src/App.tsx

```diff
@@ -1,61 +1,94 @@
+import { FC, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
The `usersFromServer` data should not be imported directly here. According to the task requirements, data should be prepared in the `App` component and passed down to child components. Consider fetching or preparing this data within the `App` component and then passing it to `TodoList` or any other relevant child components.